namer/consul/src/main/scala/io/buoyant/namer/consul/ConsulInitializer.scala

```diff
@@ -60,6 +61,7 @@ case class ConsulConfig(
   @JsonIgnore
   def newNamer(params: Stack.Params): Namer = {
     val service = Http.client
+      .withStack(Http.client.stack.remove(Retries.Role))
```

Comment:
This is `Retries.moduleRequeueable` which is the requeues module, included by the stack client by default.